cwms-data-api/src/main/java/cwms/cda/data/dto/texttimeseries/RegularTextTimeSeriesRow.java

```diff
@@ -0,0 +1,223 @@
+package cwms.cda.data.dto.texttimeseries;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.annotation.JsonPOJOBuilder;
+import hec.data.timeSeriesText.DateDateKey;
+import hec.data.timeSeriesText.TextTimeSeriesRow;
+import java.math.BigDecimal;
+import java.util.Date;
+
+@JsonDeserialize(builder = RegularTextTimeSeriesRow.Builder.class)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
+public class RegularTextTimeSeriesRow implements TextTimeSeriesRow {
+
+    private final Date dateTime;
+    private final Date versionDate;
+    private final Date dataEntryDate;
+    private final Long attribute;
+    private final String textId;
+    private final String textValue;
+    private final boolean newData;
```

Comment:
Pretty sure `newData` is just used by REGI as a modified flag

Reply:
Thank you! That has been driving me crazy